tests/sgx_zerobase/enc/enc.cpp

```diff
@@ -3,42 +3,87 @@
 
 #include <openenclave/enclave.h>
 #include <openenclave/internal/tests.h>
+#include <iostream>
+
 #include "sgx_zerobase_t.h"
 
-const char* protected_message = "Hello world from Enclave\n\0";
+static uint64_t faulting_address;
```

Comment:
Since we are testing for address zero as well, it is better to initialize `faulting_address` to a non-zero invalid value like `(uint64_t) -1`.